libr/asm/arch/gb/gbdis.c

```diff
@@ -25,25 +25,30 @@ static int gbOpLength(int gboptype){
 #ifndef GB_DIS_LEN_ONLY
 static int gbDisass(RAsmOp *op, const ut8 *buf, int len){
 	int foo = gbOpLength (gb_op[buf[0]].type);
-	if (len<foo)
+	if (len < foo) {
 		return 0;
+	}
+	const char *buf_asm = NULL;
 	switch (gb_op[buf[0]].type) {
 	case GB_8BIT:
-		sprintf (op->buf_asm, "%s", gb_op[buf[0]].name);
+		buf_asm = sdb_fmt ("%s", gb_op[buf[0]].name);
```

Comment:
Maybe to make an define to `sdb_fmt` with r2 prefix? Something like `r_util_fmt`? Because it feels out of context to use database function here.